src/main/resources/application.yaml

```diff
@@ -218,3 +218,6 @@ process-payments:
   enabled: ${PROCESS_PAYMENTS_ENABLED:true}
 
 use-wrapping-zip: true
+
+ocr-validation:
+  retry-count: 2
```

Comment:
do you plan to add any other properties under `ocr-validation`?

Reply:
how many times will it try to do the validation 2 or 3?

Reply:
2 total tries

Reply:
and there will be no other props in ocr-validation - as of now, so yes, will make it flat